nce/NCE-Pairwise-SM/trec_dataset.py

```diff
@@ -0,0 +1,31 @@
+from torchtext import data
+import os
+
+class TrecDataset(data.TabularDataset):
+    dirname = 'data'
+    @classmethod
+
+    def splits(cls, question_id, question_field, answer_id, answer_field, external_field, label_field,
+               train='train.tsv', validation='dev.tsv', test='test.tsv'):
+        path = './data'
```

Comment:
in the new version of torchtext, the splits method is updated:
```python 
def splits(cls, question_id, question_field, answer_id, answer_field, external_field, label_field, root='.data',
               train='trecqa.train.tsv', validation='trecqa.dev.tsv', test='trecqa.test.tsv'):
        path = './data'
        return super(TrecDataset, cls).splits(
            path, root, train, validation, test,...
```

Reply:
@Victor0118 did you run this on the latest torchtext version? if so, `root` is one of the args to `splits()` method. Please double check this.